modules/appeals_api/app/models/appeals_api/higher_level_review.rb

```diff
@@ -48,7 +48,13 @@ def pdf_structure(version)
       ).new(self)
     end
 
-    # 1. VETERAN'S NAME
+    def claimant
+      # TODO: Consider how we want to handle incomplete claimant headers and form data - this is not necessarily ideal
```

Comment:
Please consider and provide feedback here :)

Reply:
Looking good, nice work!

Hmm, what do you think about using the null object pattern here? That way we wouldn't have to litter the form data and structure classes with conditional logic...

Reply:
Yes! I'll work on it now